sharethis.libraries.yml

```diff
@@ -25,3 +25,15 @@ drupal.sharethispickerexternal:
   dependencies:
       - core/jquery
 
+drupal.sharethispickerexternalbuttonsws:
```

Comment:
why library names are drupal.* ? is there any specific reasons for it ? why not simpler name ? like sharethispickerexternalbuttonsws ?